Feat/13 create footer

Mariah0-0 opened the pull request:
Closes #13 

Developed footer
Added links to social media
Added email input field for subscriptions
Included client side email validation

dacoderr reviewed (approved):
updated container

Mariah0-0 committed:
feat(assets): add assets folder and footer images

Mariah0-0 committed:
feat(footer): add footer with social media links and subscription input field

Mariah0-0 committed:
chore(assets): remove unused files from assets folder

Mariah0-0 committed:
feat(footer): add email validation using regex

hybachi reviewed (changes requested):
Good work, the Footer looks best on mobile devices but for larger devices you need to scale it up and space things out accordingly.

Changes Requested:
- Increase link size on mobile screen and scale it up accordingly for all screen sizes
- Add link hover/click effects (color change) and button effects
- Align all columns of the footer at the top baseline and increase spacing between columns as screen size gets larger

Mariah0-0 committed:
fix(footer): improve footer layout

Increase font size.
Add medium device breakpoint.
Align sections to the top.
Fix hover and click effects.

hybachi reviewed (approved):
Yes, good work!
